spec/dummy/config/boot.rb

```diff
@@ -7,4 +7,6 @@
   Bundler.setup
 end
 
-$:.unshift File.expand_path('../../../lib', __dir__)
\ No newline at end of file
+$:.unshift File.expand_path('../../../lib', __dir__)
+
+require "logger" # Fix concurrent-ruby removing logger dependency which Rails itself does not have
```

Comment:
While there are more RuboCop violations, let's not introduce a new one :)
```suggestion
require 'logger' # Fix concurrent-ruby removing logger dependency which Rails itself does not have
```